tests/test_model.py

```diff
@@ -1539,16 +1584,53 @@ def test_add_load_by_geometry_ids(self, create_default_3d_soil_material: SoilMat
         npt.assert_allclose(model.process_model_parts[2].parameters.value, expected_2d_model_part_parameters.value)
         npt.assert_allclose(model.process_model_parts[2].parameters.active, expected_2d_model_part_parameters.active)
 
-        # check raising of errors
+        # add boundary conditions in 0d, 1d and 2d
+        model.add_load_by_geometry_ids([], create_default_point_load_parameters, "point_loads")
```

Comment:
you don't assert anything after this adding of load, also the comment is not correct.

also moving load is not tested